scripts/build-openapi.js

```diff
@@ -1,113 +1,51 @@
 #!/usr/bin/env node
 
 const fs = require("fs")
-const OAS = require("oas-normalize")
-const swaggerInline = require("swagger-inline")
-const { exec } = require("child_process");
+const path = require("path")
+const { exec } = require("child_process")
+const yaml = require("js-yaml")
 
-const isDryRun = process.argv.indexOf('--dry-run') !== -1;
+const run = () => {
+  for (const apiType of ["store", "admin"]) {
+    const inputJsonFile = path.resolve(
+      __dirname,
+      "../",
+      `packages/medusa/dist/oas/${apiType}.oas.json`
+    )
+    const outputJsonFile = path.resolve(
+      __dirname,
+      "../",
+      `docs/api/${apiType}-spec3.json`
+    )
+    const outputYamlFile = path.resolve(
+      __dirname,
+      "../",
+      `docs/api/${apiType}-spec3.yaml`
+    )
 
-// Storefront API
-swaggerInline(
-  ["./packages/medusa/src/models", "./packages/medusa/src/api/middlewares" , "./packages/medusa/src/api/routes/store"],
-  {
-    base: "./docs/api/store-spec3-base.yaml",
+    fs.copyFileSync(inputJsonFile, outputJsonFile)
```

Comment:
thought: Should we also use the promise API of the fs? what is your opinion?

Reply:
Done